### PR TITLE
Update previous release Crystal 1.1.0

### PR DESCRIPTION
**Pull request by straight-shoota**
Still waiting on 1.1.0 docker images

### CODE REVIEW

**Comment by straight-shoota** [comment]
Specs are failing due to #10961, but that's fine (for this PR).

File: src/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-1.1.0
+1.2.0-dev
```
Comment (asterite) [inline src/VERSION:1]: Should this be 1.1.1? I think we have at least one fix for that version.

Comment (straight-shoota) [reply src/VERSION:1]: The master branch should be heading towards the next minor release. We shouldn't keep it frozen any longer (it's been a month now).

If we need patch releases, they would be backported to the `release/1.1` branch.
What fix do you have in mind?

Comment (straight-shoota) [reply src/VERSION:1]: Ah, I guess #10960?

Comment (asterite) [reply src/VERSION:1]: Yes, that one. But I can't remember how branches and releases work, so better to ignore me 😅

File: shell.nix [hunk 2]
```diff
@@ -52,17 +52,17 @@ let
   # Hashes obtained using `nix-prefetch-url --unpack <url>`
   latestCrystalBinary = genericBinary ({
     x86_64-darwin = {
-      url = "https://github.com/crystal-lang/crystal/releases/download/1.0.0/crystal-1.0.0-1-darwin-x86_64.tar.gz";
+      url = "https://github.com/crystal-lang/crystal/releases/download/1.1.0/crystal-1.1.0-1-darwin-x86_64.tar.gz";
       sha256 = "sha256:1ff05f7v31r7xw4xk1a5zns77k3hrgdb9cn15w2zsps83iqlq81i";
```
Comment (bcardiff) [inline shell.nix:56]:
```suggestion
      sha256 = "sha256:0dk893g5v3y11hfmr6viskhajnlippwcs8ra8azxa9rjh47lx8zg";
```

File: shell.nix [hunk 3]
```diff
@@ -52,17 +52,17 @@ let
...
     };
 
     x86_64-linux = {
-      url = "https://github.com/crystal-lang/crystal/releases/download/1.0.0/crystal-1.0.0-1-linux-x86_64.tar.gz";
+      url = "https://github.com/crystal-lang/crystal/releases/download/1.1.0/crystal-1.1.0-1-linux-x86_64.tar.gz";
       sha256 = "sha256:13940gjs1zl29wrhngzylhckxgzb8xh16bniqik5lslp6qpljqy4";
```
Comment (bcardiff) [inline shell.nix:61]:
```suggestion
      sha256 = "sha256:1n967087p0km0v4pr7xyl4gg5cfl1zap7kas94gw4cs4a90irwgd";
```

File: shell.nix [hunk 4]
```diff
@@ -52,17 +52,17 @@ let
...
     };
 
     i686-linux = {
-      url = "https://github.com/crystal-lang/crystal/releases/download/1.0.0/crystal-1.0.0-1-linux-i686.tar.gz";
+      url = "https://github.com/crystal-lang/crystal/releases/download/1.1.0/crystal-1.1.0-1-linux-i686.tar.gz";
       sha256 = "sha256:18xg2nxg68cx0ngidpzy68wa5zqmcz0xfm0im5sg8j8bnj8ccg35";
```
Comment (bcardiff) [inline shell.nix:66]:
```suggestion
      sha256 = "sha256:06qzhrq4la7fkk1y6nr5kq52gxfnrlbnh9lg7ppbxqglr39ygml3";
```

**Review by bcardiff — CHANGES_REQUESTED** [review]
The shas in shell.nix needs to be updated `$ nix-prefetch-url --unpack <url>`

Other than that LGTM

**Review by bcardiff — APPROVED** [review]

**Comment by straight-shoota** [comment]
There are still a lot of CI failures, even after #10964 has been merged.

* `windows-job` fails because of a regression introduced in #10960. The spec should just be disabled on windows ( #10971). 
* `test_alpine` hits an invalid memory access somewhere in compiler specs. Need to investigate where and why. Might be related to #10964.
* aarch64 `musl-build` has no useful debug information. aarch64 jobs have been failing a lot, lately. `gnu-build` succeeded at least, though. We need to look into that.

I think we should still merge this. Red CI isn't a huge issue (especially at the beginning of a release cycle) and it helps avoid regressions like  #10971.

**Review by bcardiff — APPROVED** [review]

**Pull request merged**